docs/api-reference/classes/SearchEngine/buildSql.md

```diff
@@ -1,15 +1,19 @@
 ---
-uid: crmscript_ref_SearchEngine_buildSql
+uid: crmscript_class_searchengine_buildsql
 title: SearchEngine.buildSql()
+description: CRMScript method in the SearchEngine class that returns the generated SQL query 
 intellisense: SearchEngine.buildSql
-keywords: buildSql()
+keywords: buildSql(), buildSql(String)
 so.topic: reference
 ---
 
 
-Returns the SQL query generated by the SearchEngine. The query is built by `addCriteria()`, `addComparison()`, `addData()` etc.
+Returns the SQL query generated by the SearchEngine. The query is built by `addCriteria()`, `addComparison()`, `addData()`, and so on.
 
+> [!NOTE]
+> For versions 7.0 and newer, the statement returned will be an estimate of the SQL, since the query is sent to NetServer instead.
 
-Note that when running version versions from 7.0, the statement returned will be an estimate of the SQL, since the query is sent to NetServer instead.
-
+## Methods
 
+* SearchEngine.buildSql()
+* SearchEngine.buildSql(String s)
```

Comment:
String parameter description: Determines type of query to build. 
Parameter options:
"select"
"insert"
"update"
"delete"